portfolio/src/main/webapp/photo.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Photo Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1>Welcome to Sunflower's Photo Gallery</h1>
+      <p>Here's a collection of my favorite photos. Enjoy!
+      <div class="image-thumbnails">
```

Comment:
Please format these elements accordingly. https://google.github.io/styleguide/htmlcssguide.html#HTML_Formatting_Rules.

Reply:
I'm a bit not sure how to change this. I did not intend to make <div class="image-thumbnails"> a child of <p>, will adding an end tag </p> on line 11 help clarify the structure?

Reply:
I'm a bit not sure how to change this. I did not intend to make ``<div class="image-thumbnails">`` a child of ``<p>``, will adding an end tag ``</p>`` on line 11 help clarify the structure?

Reply:
Actually I mean
```
<a ...> <img ...> </a>
```
or
```
<a ...>
  <img ...>
</a>
```